rasa/train.py

```diff
@@ -164,19 +164,20 @@ def train(
             kwargs=kwargs,
         )
 
+    new_fingerprint = await model.model_fingerprint(file_importer)
     old_model = model.get_latest_model(output_path)
-    retrain_core, retrain_nlu = model.should_retrain(
-        new_fingerprint, old_model, train_path
+    retrain = ShouldRetrain(
+        core=False, nlu=False, nlg=False, force_train=force_training
```

Comment:
can we provide default values for `ShouldRetrain` so that we can just do `ShouldRetrain`?